npm install fails - invalid xero-node path in package-lock.json in master

sesh@IMCHLT359:~/xero-node-oauth2-app$ which node
/home/sesh/.nvm/versions/node/v12.12.0/bin/node
sesh@IMCHLT359:~/xero-node-oauth2-app$ npm i
npm WARN tarball tarball data for xero-node@4.12.0 (sha512-OYXAZvOnT8/FsUF3ioc6NvCh9tj6oH8xzj65AEqeKXL2FojtLzzkyxXICjNiDE3lcZ9BrS612iUPuzctrNHupg==) seems to be corrupted. Trying one more time.
npm WARN tarball tarball data for xero-node@4.12.0 (sha512-OYXAZvOnT8/FsUF3ioc6NvCh9tj6oH8xzj65AEqeKXL2FojtLzzkyxXICjNiDE3lcZ9BrS612iUPuzctrNHupg==) seems to be corrupted. Trying one more time.
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/projects/models.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/files/objectGroup.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/files/objectType.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/onlineInvoice.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/onlineInvoices.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/payroll-au/openingBalances.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/organisation.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/organisations.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/overpayment.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/overpayments.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/assets/pagination.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/bankfeeds/pagination.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/payroll-nz/pagination.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/projects/pagination.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/payroll-au/payItem.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/payroll-au/payItems.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/payment.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/paymentDelete.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/payroll-au/paymentFrequencyType.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/payroll-nz/paymentLine.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/payroll-nz/paymentMethod.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/payroll-nz/paymentMethodObject.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/payments.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/paymentService.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/paymentServices.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/paymentTerm.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/model/accounting/paymentTermType.js.map'
npm WARN tar ENOENT: no such file or directory, open '/home/sesh/xero-node-oauth2-app/node_modules/.staging/xero-node-536913f3/dist/gen/api/payrollAUApi.js.map'
npm WARN xero-node-oauth2-app@1.0.0 No repository field.
npm WARN The package fs-extra is included as both a dev and production dependency.

npm ERR! code EINTEGRITY
npm ERR! Verification failed while extracting xero-node@4.12.0:
npm ERR! Verification failed while extracting xero-node@4.12.0:
npm ERR! sha512-OYXAZvOnT8/FsUF3ioc6NvCh9tj6oH8xzj65AEqeKXL2FojtLzzkyxXICjNiDE3lcZ9BrS612iUPuzctrNHupg== integrity checksum failed when using sha512: wanted sha512-OYXAZvOnT8/FsUF3ioc6NvCh9tj6oH8xzj65AEqeKXL2FojtLzzkyxXICjNiDE3lcZ9BrS612iUPuzctrNHupg== but got sha512-aTTbvawQ9zxxxCYY/mTDZSWQbhZnHXoSXKHJ8wyV3FmEnV17SIDFegaJujpBvOBNyOf024Z5yJYqpct6FtqWYA==. (345001 bytes)

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/sesh/.npm/_logs/2021-06-11T05_27_10_136Z-debug.log

--- a/src/app.ts
+++ b/src/app.ts
@@ -1308,8 +1308,8 @@ class App {
           0,
           true,
           false,
-          true,
           4,
+          true,
           {
             headers: {
               'contentType': 'application/json'

